.solhint.json

```diff
@@ -4,7 +4,7 @@
     "indent": ["error", 4],
     "quotes": ["error", "double"],
     "max-line-length": ["error", 129],
-    "function-max-lines": ["error", 61],
+    "function-max-lines": ["error", 65],
```

Comment:
I really dislike that we keep pushing this value in each PR. That kind of makes this rule pointless. Can we just remove the max lines constraint?

Reply:
YES!